packages/expo-secure-store/android/src/main/java/expo/modules/securestore/PostEncryptionCallback.kt

```diff
@@ -0,0 +1,11 @@
+package expo.modules.securestore
+
+import expo.modules.core.Promise
+import org.json.JSONException
+import java.security.GeneralSecurityException
+
+// Interface used to pass logic that needs to happen after encryption/decryption
+interface PostEncryptionCallback {
```

Comment:
```suggestion
fun interface PostEncryptionCallback {
```
Using [`fun` (SAM) interfaces](https://kotlinlang.org/docs/fun-interfaces.html) will gain you a bunch of advantages - see my other comments

Reply:
Done in https://github.com/expo/expo/pull/14512/commits/531d1a47ddd80d2b5758771176f205a0c590ca9a.